src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -46,13 +56,20 @@
   private static final String DEFAULT_STRING = "";
   private static final String USER_ENTITY = "User";
   private static final String USER_BIO_PROPERTY = "bio";
+  private static final String USER_BLOBKEYS_PROPERTY = "blobkeys";
   private static final String USER_EMAIL_PROPERTY = "email";
   private static final String USER_FOUND_PROPERTY = "user-found";
   private static final String USER_FRIENDS_LIST_PROPERTY = "friends-list";
   private static final String USER_ID_PROPERTY = "id";
   private static final String USER_NAME_PROPERTY = "name";
-
-  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final String USER_PHOTO_1_PROPERTY = "profile-photo";
+  private static final String USER_PHOTO_2_PROPERTY = "photo-2";
+  private static final String USER_PHOTO_3_PROPERTY = "photo-3";
+  private static final String USER_PHOTO_4_PROPERTY = "photo-4";
+  private static final String USER_PHOTO_5_PROPERTY = "photo-5";
+
+  BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
nit: add a @VisibleForTesting annotation to each of these fields

(or if that annotation isn't available, just add a comment noting that they are visible only for testing)